cot/src/db/migrations.rs

```diff
@@ -1330,7 +1330,7 @@ impl MigrationDependency {
     /// creates the model with the given app and table name before the current
     /// migration.
     #[must_use]
-    pub const fn model(app: &'static str, table_name: &'static str) -> Self {
```

Comment:
Can we have this change as a separate PR (along with the `APP_NAME` const, etc.)? I think we should merge this fix ASAP, while the rest of this can wait until we're a bit closer to v0.2.

Reply:
Done in https://github.com/cot-rs/cot/pull/228